cypress/support/hostsfunctions.ts

```diff
@@ -64,7 +64,26 @@ function editHost(inventoryID: number, host_type: string, hostName: string) {
   cy.hasDetail(/^Description$/, 'This is the description edited');
 }
 
-function deleteHost(inventoryID: number, host_type: string, hostName: string) {
+function editHostDetailsView(inventoryID: number, host_type: string, hostName: string) {
+  // function that editing host data
+  // this function cover both inventory host and stand alone host
+  let url = '';
+  if (host_type === 'inventory_host') {
+    url = `/infrastructure/inventories/inventory/${inventoryID}/hosts/?page=1&perPage=10&sort=name`;
+  } else {
+    url = '/infrastructure/hosts?page=1&perPage=10&sort=name';
+  }
+  cy.visit(url);
```

Comment:
We should not be using `cy.visit` and we should not be hard coding url paths in the tests
please use `cy.navigate`, search and filter the resource, navigate to its details page or list page

Reply:
I'll change it in the next PR since the url call is in function there

Reply:
done in : https://github.com/ansible/ansible-ui/pull/2491